.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
- - [ ] Closes #xxxx
- - [ ] Tests added / passed
- - [ ] Passes ``git diff upstream/master **/*py | flake8 --diff``
- - [ ] Fully documented, including `whats-new.rst` for all changes and `api.rst` for new API
+ - [ ] Closes #xxxx (remove if there is no corresponding issue, which should only be the case for minor changes)
+ - [ ] Tests added / passed (remove if this change only consists of documentation or performance improvements)
```

Comment:
probably drop the performance improvements part. In those cases, we at least will want the tests to pass.